tests/CalculatorTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: JEDV
+ * Date: 04/11/2020
+ * Time: 22:19
+ */
+
+namespace Application\Tests;
```

Comment:
```suggestion
namespace Application\Tests;

```
quality check un espacio entre la declarion del namespace y los use